titleize/titleize.js

```diff
@@ -1,8 +1,14 @@
+const performTitleCase = sentence => {
+  return sentence.charAt(0).toUpperCase() + sentence.slice(1).toLowerCase();
+}
 
-function titleize(...args) {
-  return args;
+function titleize(sentence) {
+  if (!sentence.split) return sentence;
+  let result = [];
+  sentence.split(" ").forEach(word => result.push(performTitleCase(word)));
```

Comment:
Using `forEach` and having side effects can be avoided here. You should consider using `for of` loop.